tests/tests_suite_test.go

```diff
@@ -337,10 +338,6 @@ func deployApp(name string) App {
 	app := App{Name: appName, URL: strings.Replace(url, "deis", appName, 1)}
 
 	cmd := createApp(app.Name)
-	Eventually(cmd).Should(SatisfyAll(
```

Comment:
good point, at least the first two `Say`'s need to go back; this is an unintended relic mid-refactor of `createApp`...